apps/myaccount/src/components/profile/profile.tsx

```diff
@@ -121,6 +124,84 @@ export const Profile: FunctionComponent<ProfileProps> = (props: ProfileProps): R
 
     const allowedScopes: string = useSelector((state: AppState) => state?.authenticationInformation?.scope);
 
+    const USER_CLAIM_UPDATE_CONNECTOR: string = "user-claim-update";
+    const ENABLE_MOBILE_VERIFICATION: string = "UserClaimUpdate.MobileNumber.EnableVerification";
+    const ENABLE_EMAIL_VERIFICATION: string = "UserClaimUpdate.Email.EnableVerification";
```

Comment:
Since we have already imported https://github.com/wso2/identity-apps/pull/6220/files#diff-cf121fdf7ae615622f88ac45f4b70c77e03017fcf5cd222e0847ce4b7f6ddec1R19, moved the constant there.